src/components/BarChart/BarChart.jsx

```diff
@@ -335,6 +358,46 @@ const BarChart = createClass({
 			? yAxisTooltipDataFormatter
 			: yAxisFinalFormatter;
 
+		if (_.isEmpty(data) || width < 1 || height < 1 || isLoading) {
+			const emptyMessageBodyProp = _.get(getFirst(this.props, BarChart.EmptyMessageBody), 'props');
+			const emptyMessageTitleProp = _.get(getFirst(this.props, BarChart.EmptyMessageTitle), 'props', {children: 'You have no data.'});
+
+			return (
+				<EmptyStateWrapper
+					isEmpty={_.isEmpty(data)}
+					isLoading={isLoading}
+				>
+					<EmptyStateWrapper.EmptyMessageBody {...emptyMessageBodyProp}/>
```

Comment:
see comment in LineChart: https://github.com/appnexus/lucid/pull/571/files?diff=split#r83692560